packages/ember-htmlbars/lib/helpers/if_unless.js

```diff
@@ -64,6 +64,7 @@ function appendBlockConditional(view, inverted, helperName, params, hash, option
     view.appendChild(BoundIfView, {
       _morph: options.morph,
       _context: get(view, 'context'),
+      renderer: get(view, 'renderer'),
```

Comment:
`createChildView` feels like a better place for this than `appendChild`, what say you all?